src/WebJobs.Script/Settings/ISettingsManager.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Settings
+{
+    public interface ISettingsManager
+    {
+        void Clear();
+        string GetEnvironmentSetting(string environmentSettingKey);
+        void SetEnvironmentSetting(string environmentSettingKey, string environmentSettingValue);
```

Comment:
Could we just make this `GetSetting` and `SetSetting`? Use of `environment` in all these properties and parameters probably isn't really necessary.